### PR TITLE
Small UI improvements (please see description for list of tasks)

### LINKED ISSUE #149: `<ol>` cannot appear as a descendant of `<p>`
image
image

### LINKED ISSUE #162: Add a little more space in between cards or make it easier to distinguish between the different cards in MyNiwa
Image from iOS (4)

### LINKED ISSUE #171: Make splash screen logo + "Niwa" kanji image smaller
You have to scroll to get to the button to go to your niwa.

Simulator Screen Shot - iPhone 12 Pro Max - 2021-09-11 at 08 50 54

### PR DESCRIPTION
**Pull request by kome12**
## Description

- Add spacing in between cards in MyNiwa
- Fix `ol` error in about page
- Fix splash page so scroll is not necessary to see button

## Closes issue(s)

Closes #162
Closes #149 
Closes #171 

## Screenshots

スクリーンショット 2021-09-11 11 33 07

スクリーンショット 2021-09-11 11 27 47

splashpage

## Changes include

- [X] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [X] I have tested this code
- [ ] Add here the checks for this PR. If any please delete these checks

### CODE REVIEW

File: src/pages/About/index.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import CardActions from "@material-ui/core/CardActions";
 import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
+import { makeStyles } from "@material-ui/core/styles";
 import Typography from "@material-ui/core/Typography";
+import { motion } from "framer-motion";
+import { useHistory } from "react-router";
+import { SignIn } from "../../components/SignIn";
+import { useUserState } from "../../store/user/useUserState";
+import "./About.css";
```
Comment (daiHash) [inline src/pages/About/index.tsx:13]: If you can change for this component to use styles as CSS modules 🙏

**Review by daiHash — APPROVED** [review]
Nice LGTM!👍

File: src/pages/About/index.tsx (hunk 1 above)
Comment (kome12) [reply src/pages/About/index.tsx]: Updated to use CSS module!

File: src/pages/About/index.tsx [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 import CardActions from "@material-ui/core/CardActions";
 import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
+import { makeStyles } from "@material-ui/core/styles";
 import Typography from "@material-ui/core/Typography";
+import { motion } from "framer-motion";
+import { useHistory } from "react-router";
+import { SignIn } from "../../components/SignIn";
+import { useUserState } from "../../store/user/useUserState";
+import styles from "./About.module.css";
```
Comment (daiHash) [inline src/pages/About/index.tsx:13]: Thanks! LGTM

**Review by daiHash — APPROVED** [review]

**Pull request merged**